### ISSUE TITLE
Set the `status-firefox###` flag to `affected` when the Intermittent Failures Robot reports recent failures

### ISSUE THREAD

**Issue by cpeterso** [mozilla/relman-auto-nag#1657]
The Intermittent Failures Robot adds "X failures in Y pushes were associated with this bug in the last Z days" bug comments.

Suggestion: to help track affected versions, the Intermittent Failures Robot or autonag could also set the bug's `status-firefox###` flag to `affected` (unless the current status is `wontfix` or `fix-optional`, indicating that someone knows that version is affected but decided that it doesn't require a fix).

Old `status-firefox###` value|New `status-firefox###` value
--|--
`---`|`affected`
`?`|`affected`
`unaffected`|`affected`
`affected`|*(no change needed: we already know this version is affected)*
`fix-optional`|*(no change needed: someone already knows this version is affected)*
`fixed`|`affected`
`wontfix`|*(no change needed: someone already knows this version is affected*
`verified`|`affected`
`disabled`|`affected`
`verified disabled`|`affected`

**Comment by Archaeopteryx** [comment]
* The bot might comment after the bug got fixed, e.g. during the last week. To know if the failures have been observed after this fix landed, the push datetimes and version number need to be checked. This affects status changes to fixed and verified.
* In a similar vein the repository/branch has to be taken into account to identify the correct version number to modify.
To keep mind: On mozilla-central to mozilla-beta merge days, the version number on autoland is incorrect. The version gets increase on central and some changesets landed on autoland while the old version number was active (after the last merge with the old version number and before the version increase got added to central).
The post-fix failure identification likely requires the most work.

**Comment by mathewhodson** [comment]
I don't think it makes sense to change the flag from `disabled` to `affected`. `disabled` is often set if it's a Nightly only crash or something that happens when an off by default feature is enabled. I usually see it set when a bug is mitigated by one of those things but there is still an intention to get a more complete fix committed.

**Comment by cpeterso** [comment]
> I don't think it makes sense to change the flag from `disabled` to `affected`. `disabled` is often set if it's a Nightly only crash or something that happens when an off by default feature is enabled.

SGTM. I updated the table above to say "(no change needed: test failure for a feature that is disabled for users)" for the `disabled` and `verified disabled` statuses.
